.prettierignore

```diff
@@ -8,3 +8,4 @@ package-lock.json
 **/package-lock.json
 CHANGELOG.md
 .eslintignore
+circuits/ts/verifier_groth16.sol
```

Comment:
Why? Is this in autogenerated file (verifying contract)?

Reply:
I added this one because it's not in contracts folder and I don't want to trigger prettier for now. Later I'll remove it if this file is still in circuits package.